src/types/components/CardFieldInput.ts

```diff
@@ -23,6 +23,7 @@ export namespace CardFieldInput {
     placeholderColor?: string;
     cursorColor?: string;
     textErrorColor?: string;
+    fontFamily?: string;
```

Comment:
it could be too big since on iOS are available a lot of fonts:  https://developer.apple.com/fonts/system-fonts/

Reply:
I was thinking maybe just the ones that are available across platforms, I think you said Android only had a smaller set. And then we can just allow `| string` also, wdyt? But yah, also okay to leave out, just thought might be nice for autocomplete in vscode.

Reply:
it could be misleading for Typescript users and they might think these are the only available fonts, also there are no common fonts names for platforms, android uses camelCase while ios, words after spaces

Reply:
@arekkubaczkowski well in that case we could actually need separate params then, no?

or an object `fontFamily?: { ios: string; android: string; }`

Reply:
For such cases react native provides Platform.select helper which should be applied here